### ISSUE TITLE
Options binding breaks WCAG

### ISSUE THREAD

**Issue by trubens** [knockout/knockout#1239]
The options binding breaks the Robust principle in WCAG 2. To fix it, we need to add the "selected" HTML-attribute to the selected item.

HTML CodeSniffer (http://squizlabs.github.io/HTML_CodeSniffer) reports:
```
This select element does not have a value available to an accessibility
API. Add one by adding a "selected" attribute to one of its options.

Principle: Robust
Techniques: H91
```

**Comment by SteveSanderson** [comment]
To help us focus on more highly-prioritised issues, we're tidying up our issue backlog. We're closing the oldest feature requests that don't have pull requests and score very low on our [triage ranking system](http://triage.knockoutjs.com/). The low triage score for this feature request suggests we should focus our finite resources on other issues. For more details, [see here](https://github.com/knockout/knockout/issues/1754). Thanks for posting this feature request though. If you still actively want this feature, please reopen this issue, or post a new one (and ideally, be prepared to supply a pull request if we agree the feature would be accepted). Thanks!
